python/pyspark/pandas/tests/test_dataframe.py

```diff
@@ -6076,7 +6076,13 @@ def test_corrwith(self):
     def _test_corrwith(self, psdf, psobj):
         pdf = psdf.to_pandas()
         pobj = psobj.to_pandas()
-        for method in ["pearson", "spearman", "kendall"]:
+        # Regression in pandas 1.5.0 when other is Series and method is "pearson" or "spearman"
+        # See https://github.com/pandas-dev/pandas/issues/48826
+        if LooseVersion(pd.__version__) >= LooseVersion("1.5.0") and isinstance(pobj, pd.Series):
+            methods = ["kendall"]
```

Comment:
why also skipping `pearson `? it is also changed in pandas 1.5?

Reply:
Yes, the regression is shown in both `pearson` and `spearman` 😿

Reply:
https://github.com/pandas-dev/pandas/pull/46174 FYI

Reply:
what about also adding this link together with `https://github.com/pandas-dev/pandas/issues/48826`?
 since `https://github.com/pandas-dev/pandas/issues/48826` is only related to s`pearman`

Reply:
Sounds good! Let me add